wsmaster/codenvy-sql-schema/src/main/resources/codenvy-schema/5.0.0/1.1__init.sql

```diff
@@ -0,0 +1,37 @@
+--
```

Comment:
Hi @dmytro-ndp. Nice question, for the product version 5.0.0 please use the latest milestone directory of the product. AFAIK it's 5.0.0-M9 and name the script appropriately, e.g. for current master your script must be named as _1.3__script_description.sql_.

Short explanation:  basically db migrator considers milestone versions as subversions of the main version part e.g. for the script _5.0.0-M9/1.1__script.sql_ db version will be _5.0.0.9.1.1_, which means that new scripts must be indexed as upper versions, while _5.0.0/1.1__script.sql_ is lower one and will be ignored by the migrator.

It seems that your tests passed because script schema doesn't conflict or depend on the existing schema, so it was applied before all the other available scripts.